docker-storage-setup.spec

```diff
@@ -39,6 +40,7 @@ install -d %{buildroot}%{_sysconfdir}/sysconfig/
 install -p -m 644 %{SOURCE3} %{buildroot}%{_sysconfdir}/sysconfig/docker-storage-setup
 install -d %{buildroot}/%{dsslibdir}
 install -p -m 755 %{SOURCE4} %{buildroot}/%{dsslibdir}/libdss.sh
+install -p -m 755 %{SOURCE5} %{buildroot}/%{dsslibdir}/dss-child-read-write.sh
```

Comment:
I think this should be 

`install -p -m 755 %{SOURCE5} %{buildroot}/%{dsslibdir}/dss-child-read-write`